spearhead-blocks/parts/header-home.html

```diff
@@ -0,0 +1,25 @@
+<!-- wp:group {"backgroundColor":"high-contrast","layout":{"type":"default"}} -->
+<div class="wp-block-group has-high-contrast-background-color has-background"><!-- wp:group {"style":{"spacing":{"padding":{"bottom":"var:preset|spacing|80"}}},"className":"gapless-group","layout":{"inherit":"true","type":"constrained"}} -->
+<div class="wp-block-group gapless-group" style="padding-bottom:var(--wp--preset--spacing--80)"><!-- wp:group {"align":"full","style":{"spacing":{"padding":{"bottom":"35px","top":"35px","right":"35px","left":"35px"}}},"layout":{"type":"flex","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignfull" style="padding-top:35px;padding-right:35px;padding-bottom:35px;padding-left:35px"><!-- wp:group {"layout":{"type":"flex"}} -->
+<div class="wp-block-group"><!-- wp:site-logo {"width":64} /-->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"5px"}}} -->
+<div class="wp-block-group"><!-- wp:site-title /-->
+
+<!-- wp:site-tagline /--></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"__unstableLocation":"primary","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"right","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"}}}} /--></div>
```

Comment:
I don't think this navigation block should have __unstableLocation.  It's a depreciated attribute and no navigation locations are defined.

Reply:
I removed it. :)